src/main/java/com/dreammaster/gthandler/recipes/ElectrolyzerRecipes.java

```diff
@@ -8,293 +8,491 @@
+                GT_Values.NI,
+                GT_Values.NI,
+                null,
+                144,
+                60);
+        // Spessartine
+        GT_Values.RA.addElectrolyzerRecipe(
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Spessartine, 20L),
+                GT_Values.NI,
+                GT_Values.NF,
+                Materials.Oxygen.getGas(3000L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Manganese, 3L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 5L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.SiliconDioxide, 9L),
+                GT_Values.NI,
+                GT_Values.NI,
+                GT_Values.NI,
+                null,
+                440,
+                90);
+        // Tanzanite
+        GT_Values.RA.addElectrolyzerRecipe(
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Tanzanite, 44L),
+                ItemList.Cell_Empty.get(5L),
+                GT_Values.NF,
+                Materials.Hydrogen.getGas(2000L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Calcium, 4L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 15L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.SiliconDioxide, 18L),
+                GT_OreDictUnificator.get(OrePrefixes.cell, Materials.Oxygen, 5L),
                 GT_Values.NI,
                 GT_Values.NI,
                 null,
                 280,
                 120);
+        // Potassium Feldspar
         GT_Values.RA.addElectrolyzerRecipe(
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Biotite, 16L),
-                ItemList.Cell_Empty.get(4L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.PotassiumFeldspar, 26L),
+                GT_Values.NI,
                 GT_Values.NF,
-                Materials.Fluorine.getGas(2000L),
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Potassium, 1L),
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Magnesium, 3L),
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 3L),
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.SiliconDioxide, 3L),
-                GT_OreDictUnificator.get(OrePrefixes.cell, Materials.Oxygen, 4L),
+                Materials.Oxygen.getFluid(1000L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Potassium, 2L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 5L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.SiliconDioxide, 18L),
+                GT_Values.NI,
+                GT_Values.NI,
+                GT_Values.NI,
+                null,
+                260,
+                90);
+        // Biotite
+        GT_Values.RA.addElectrolyzerRecipe(
```

Comment:
biotite formula also doesnt work out. you are conjuring up a bucket of oxygen

Reply:
fixed